lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -9,24 +9,27 @@ use Path::Class qw();
 class Genome::Model::Tools::Transcriptome::ErccMapUnaligned {
     is => 'Command::V2',
     has_input => [
+        model => {
+            is => 'Text',
```

Comment:
If this `is => 'Genome::Model'` (or a `Genome::Model::Build` if the previous suggestion is taken), then an ID (or other identifying information) can be used on the command line, and the property will validate that the object exists and `$self->model` will return it directly.